pkg/certificate/manager.go

```diff
@@ -86,23 +75,21 @@ const (
 // approve the generated cert/key with k8s certification approval mechanism
 func NewManager(
 	client client.Client,
-	webhookName string,
-	webhookType WebhookType,
-	namespace string,
-	caCertDuration time.Duration,
-	serviceCertDuration time.Duration,
+	options Options,
 ) *Manager {
 
+	options.setDefaults()
```

Comment:
maybe also add a sanity check?

Reply:
I don't want to complicate this more than needed, we can add sanity at differet PR if needed.